jenkins/map.jinja

```diff
@@ -5,9 +5,13 @@
         'netcat_pkg': 'netcat-openbsd',
         'user': 'jenkins',
         'group': 'jenkins',
-        'nginx_user': 'www-data',
-        'nginx_group': 'www-data',
+        'admin_user': 'admin',
+        'admin_pw': 'admin',
+        'nginx_user': 'nginx',
```

Comment:
@nmadhok First off, many thanks for your super fast response and review. 

So you suggest that I change 'nginx_user': 'nginx' back to 'nginx_user': 'www-data'? Is the latter needed for Debian distro?